.github/workflows/update-styles.yml

```diff
@@ -0,0 +1,55 @@
+name: Weekly Styles Update
+on:
+  workflow_dispatch:
+    inputs:
+      name:
+        description: 'Person who triggered the workflow'
+        required: true
+        default: 'Maintainer (via GitHub)'
+  schedule:
+    # Run every Wednesday
+    - cron: 0 0 * * 3
+
+jobs:
+  update-styles:
+    runs-on: ubuntu-20.04
+    name: Styles Update
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Record Date
+        id: today
+        run: |
+          echo "::set-output name=date::$(date -I)"
+          echo "::set-output name=min::$(date +%Y-%m-%d-%H-%M)"
+
+      - name: Sync lesson with carpentries/styles
+        id: update
+        uses: zkamvar/actions/update-styles@main
+        env:
+          LESSON: "Lesson Example"
+          REF: ${{ github.ref }}
+        with:
+          commit: true
+
+      - name: Create Pull Request
+        id: cpr
+        if: ${{ steps.update.outputs.update == 'true' }}
+        uses: peter-evans/create-pull-request@v3.10.0
+        with:
+          token: ${{ secrets.STYLES_WORKFLOW}}
+          commit-message: "[actions] Sync lesson with carpentries/styles"
+          branch: "update-styles-${{ steps.today.outputs.min }}"
+          delete-branch: true
+          title: "Update Styles (${{ steps.today.outputs.date }} via ${{ github.event.inputs.name }})"
```

Comment:
So, is the default message: "Update Styles (...date... via Maintainer (via GitHub))"?

Reply:
Yes, although, I guess the default really should be "Update Styles (...date...)"